core/src/main/java/me/mykindos/betterpvp/core/utilities/UtilPlayer.java

```diff
@@ -74,6 +78,19 @@ public static List<KeyValue<Player, EntityProperty>> getNearbyPlayers(Player pla
         return fetchNearbyEntityEvent.getEntities();
     }
 
+    public static boolean isPlayerFriendly (Player original, Player target) {
+        return isPlayerRelation(original, target, EntityProperty.FRIENDLY, EntityProperty.ENEMY);
+    }
+
+    public static boolean isPlayerRelation(Player original, Player target, EntityProperty compareProperty, EntityProperty defaultProperty) {
+        if (compareProperty.equals(EntityProperty.ALL)) {
+            return true;
+        }
+        GetPlayerPropertyEvent getPlayerPropertyEvent = new GetPlayerPropertyEvent(original, target, compareProperty, defaultProperty);
```

Comment:
Can be made into

GetPlayerPropertyEvent getPlayerPropertyEvent = UtilServer.callEvent(new GetPlayerPropertyEvent(original, target, compareProperty, defaultProperty));

Reply:
Implemented (with new event)